runtime/wasm/src/host_exports.rs

```diff
@@ -579,12 +499,8 @@ impl HostExports {
         x.to_string()
     }
 
-    pub(crate) fn big_decimal_from_string(
-        &self,
-        s: String,
-    ) -> Result<BigDecimal, HostExportError<impl ExportError>> {
-        BigDecimal::from_str(&s)
-            .map_err(|e| HostExportError(format!("failed to parse BigDecimal: {}", e)))
+    pub(crate) fn big_decimal_from_string(&self, s: String) -> Result<BigDecimal, anyhow::Error> {
+        BigDecimal::from_str(&s).context("failed to parse BigDecimal")
```

Comment:
This used to include the string in the error message, which is useful and more consistent with the other parse functions. Why was it removed?

Reply:
This did not include the string, it interpolated the error which is now included as a cause. But it's a good idea to include the string, I'll do that.